waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/exchange/OpenAPICommons.kt

```diff
@@ -1,5 +1,6 @@
 package id.walt.webwallet.web.controllers.exchange
```

Comment:
not part of this pr, but maybe worth renaming
- `rootPath` to smth. like `exchangeRootPath` 
- and `route` to `exchangeRoute`